components/layout/sequential.rs

```diff
@@ -23,7 +24,7 @@ pub use style::sequential::traverse_dom;
 pub fn resolve_generated_content(root: &mut Flow, layout_context: &LayoutContext) {
     fn doit(flow: &mut Flow, level: u32, traversal: &mut ResolveGeneratedContent) {
         if !traversal.should_process(flow) {
-            return
+            return;
```

Comment:
It'd be nice to move the purely-stylistic changes to another commit so it's easier to review.